components/TopNav.tsx

```diff
@@ -1,14 +1,24 @@
 import React, { useEffect, useState } from "react";
 import GitHubButton from "react-github-btn";
+import Link from "next/link";
+import { useRouter } from 'next/router';
 
 const TopNav = () => {
+  const router = useRouter();
   const [windowWidth, setWindowWidth] = useState<number>(0);
   useEffect(() => {
     setWindowWidth(window.innerWidth);
   }, []);
 
   return (
     <div className="h-16 mb-10 w-full px-3 border-b-2 border-slate-700 text-white flex justify-between items-center">
+     {router.pathname !== '/' && (
+      <div style={{  width:"100px", height:"30px",backgroundColor:"#1E293B",borderRadius:"10px",display: 'flex', justifyContent: 'center', alignItems: 'center' }} className="hover:border border-white">
```

Comment:
we have to use only tailwind classes
```suggestion
      <div className="flex justify-center items-center w-[100px] h-[30px] bg-[#1E293B] rounded-xl">
```